<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bin Pixels</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../VSOP.qrc">
    <normaloff>:/icons/iconApplication.png</normaloff>:/icons/iconApplication.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Bin to ...</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton2x2">
        <property name="text">
         <string>2 x 2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton3x3">
        <property name="text">
         <string>3 x 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../VSOP.qrc">
       <normaloff>:/icons/icons/cancel.png</normaloff>:/icons/icons/cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonContinue">
     <property name="text">
      <string>Continue</string>
     </property>
     <property name="icon">
      <iconset resource="../VSOP.qrc">
       <normaloff>:/icons/icons/accept.png</normaloff>:/icons/icons/accept.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../VSOP.qrc"/>
 </resources>
 <connections/>
</ui>
